dockerfiles/Caffe2OnnxDockerfile

```diff
@@ -0,0 +1,24 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && conda install -c anaconda cloudpickle=0.5.2 \
+      && conda install -c ezyang onnx\
+      && conda install -c conda-forge protobuf==3.4.0 \
+      && conda install -c ezyang/label/devgpu caffe2 \
+      && conda install -c ezyang onnx-caffe2 \
+      && conda install -c jjh_pytorch pytorch \
+      && conda install -c jjh_pytorch torchvision
```

Comment:
install pytorch and torchvision from the official channel: `conda install pytorch torchvision -c pytorch`